spring-integration-debezium/src/main/java/org/springframework/integration/debezium/inbound/DebeziumMessageProducer.java

```diff
@@ -213,19 +202,6 @@ protected void doStop() {
 		}
 	}
 
-	@Override
-	public void destroy() {
```

Comment:
Aren't you supposed to gracefully stop the task executor on exit?

Reply:
See a `destroy()` impl in the super class. It does call that `stop();` for us.